services/todos.js

```diff
@@ -47,49 +58,62 @@ function validateFindTasksFromTodoId(searchData){
     }else{
         throw new BadRequestError("searchData is missing")
     }
-
 }
 
 const findTasksFromTodoId = async function(searchData){
     const verifiedSearchData = validateFindTasksFromTodoId(searchData);
-    return await prisma.todos.findMany({
-        skip:verifiedSearchData.skip,
-        take:verifiedSearchData.take ,
-        orderBy: {
-            id: 'asc'
-        },
-        select:{
-            id:true,
-            name:true,
-            todo_parent_id:true,
-            description:true,
-            completed:true
-        },
-        where:{
-            todo_parent_id:verifiedSearchData.id
-        }
-    });
+
+    const query = `;SELECT todos.id as id,todos.name as name,
+     todos.description as description, todos.completed as completed,
+      todos.todo_parent_id as todo_parent_id FROM todos
+      WHERE todos.todo_parent_id=${verifiedSearchData.id} 
+        ORDER BY id ASC OFFSET ${verifiedSearchData.skip} LIMIT ${verifiedSearchData.take};`;
```

Comment:
```suggestion
    const query = `;
       SELECT
         todos.id as id,
         todos.name as name,
         todos.description as description,
         todos.completed as completed,
         todos.todo_parent_id as todo_parent_id 
      FROM todos
      WHERE todos.todo_parent_id=${verifiedSearchData.id} 
      ORDER BY id ASC
      OFFSET ${verifiedSearchData.skip}
      LIMIT ${verifiedSearchData.take}
    ;`;
```

Completamente desnecessário mudar isso, mas eu precisava te mostar quão belo é desse jeito ai xD

Reply:
Nossa EHUEHEUHEUE Ajuda BASTANTE a ler, ne (que é o objetivo do dev em uma equipe: desenvolver, também, para outro dev ).